test/unit/relayServerUtils.test.ts

```diff
@@ -0,0 +1,446 @@
+import sinon from 'sinon';
+import type {
+  RelayRequestBody,
+  EnvelopingRequestData,
+  RelayRequest,
+} from '@rsksmart/rif-relay-client';
+import { TestExchange, RelayPricer } from '@rsksmart/rif-relay-client';
+
+import { BigNumber, constants, providers } from 'ethers';
+import * as utils from '../../src/Utils';
+import { ERC20__factory, ERC20 } from '@rsksmart/rif-relay-contracts';
+import { expect, use } from 'chai';
+import * as Conversions from '../../src/Conversions';
+import { BigNumber as BigNumberJs } from 'bignumber.js';
+import { TRANSFER_HASH, TRANSFER_FROM_HASH } from '../../src/relayServerUtils';
+import { toPrecision } from '../../src/Conversions';
+import chaiAsPromised from 'chai-as-promised';
+import * as relayServerUtils from '../../src/relayServerUtils';
```

Comment:
Why couldn't we import the required functions only?
e.g:
```suggestion
import { calculateFee } from '../../src/relayServerUtils';
```

Reply:
It's because I need the relayServerUtils to stub some functions.